symplyphysics/laws/thermodynamics/gas_pressure_from_temperature.py

```diff
@@ -1,21 +1,23 @@
-from sympy import (Eq, solve)
+from sympy import Eq, solve
 from symplyphysics import (
     symbols,
     units,
     Quantity,
     Symbol,
-    print_expression,
     validate_input,
     validate_output,
+    quantities,
 )
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.thermodynamics import volume_is_constant as isochoric_law
 
 # Description
 ## The change in gas pressure when the temperature changes relative to 273 kelvin depends on temperature,
 ## standard pressure and thermal coefficient.
 ## https://www.fxyz.ru/формулы_по_физике/термодинамика_теория_теплоты/тепловое_расширение/расширение_газа/термический_коэффициент_давления/
 
-## Law is: p = p0 * y * T, where
-## p - pressure change,
+## Law is: dp = p0 * (y * T - 1), where
```

Comment:
You didn't because the original laws contain temperature in _Celsius_ units. That's why it was in such a simple form $`\Delta p = p_0 \beta t`$. Here it is true that if $`t = 0 \degree \mathrm{C}`$, then $`\Delta p = 0`$. But we use _Kelvin_ units, and $`t`$ must be replaced by $`T - T_0`$.